### DISCUSSION TITLE
How can I create structured data on a Docusaurus site?

### DISCUSSION THREAD

**Discussion by Alxira5** [facebook/docusaurus#9961] [Q&A]
Hello developers, today I want to ask you a small question about Docusaurus [SEO](https://docusaurus.io/docs/seo), is the structured data only what is shown in the documentation example or does it only work on certain pages? Since when trying to imitate the example in the configuration file, since when trying to imitate the example in the configuration file.

Here is an example of the code present in the configuration file:

```typescript
  themeConfig: {
    headTags: [
      {
        tagName: 'script',
        attributes: {
          type: 'application/ld+json',
        },
        innerHTML: JSON.stringify({
          '@context': 'https://schema.org',
          '@type': 'Person',
          image: 'img/alxira5.jpg',
          jobTitle: 'Software Developer',
          name: 'Guillermo Brito',
          url: 'https://alxira5.vercel.app/',
        }),
      },
    ],
```

**Comment by johnnyreilly** [comment]
You may find some of this helpful 

https://johnnyreilly.com/how-we-fixed-my-seo#structured-data
